packages/fxa-settings/src/components/Settings/UnitRowTwoStepAuth/index.tsx

```diff
@@ -38,19 +37,12 @@ export const UnitRowTwoStepAuth = () => {
   } = account;
   const [disable2FAModalRevealed, revealDisable2FAModal, hideDisable2FAModal] =
     useBooleanState();
-  const [add2FAModalRevealed, revealAdd2FAModal, hideAdd2FAModal] =
-    useBooleanState();
   const ftlMsgResolver = useFtlMsgResolver();
 
-  const handleAdd2FA = useCallback(() => {
-    hideAdd2FAModal();
-    navigateWithQuery(route);
-  }, [navigateWithQuery, hideAdd2FAModal]);
-
-  const handleAdd2FAClick = useCallback(() => {
+  const handleAdd2FAClick = () => {
```

Comment:
Just double checking that we want to remove useCallback. This is a callback after all...